news/announcements.go

```diff
@@ -139,3 +139,14 @@ func (a Announcements) Filter(p []string) Announcements {
 	}
 	return filteredAnnouncements
 }
+
+// HTML Converts Announcements to an unordered html list.
+func (a Announcements) HTML() string {
+	var html string
+	html += "<ui>"
```

Comment:
Do you think `strings.Builder` is overkill here?  It's used [here](https://github.com/circa10a/go-aws-news/blob/master/providers/discord/discord.go#L50-L52) already so maybe for consistency?  Not a big deal either way.

Reply:
That's fair. I just pushed the change.